### PR TITLE
Some refactors

### PR DESCRIPTION
**Pull request by ddomingues**
* Fix REAME
* Update some gems
* Add spring rspec commands

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -89,6 +88,7 @@ group :test do
   gem "ffaker",                   "~> 2.2.0"
   gem "poltergeist",              "~> 1.10.0"
   gem "rails-controller-testing", "~> 0.1.1", require: false
+  gem "rspec-rails",              "~> 3.5"
```
Comment (roalcantara) [inline Gemfile]: Ok! But, why is this happening?

If we don't talk about some change, wouldn't be better if we could explain the reason for this change (providing a [source](https://github.com/rspec/rspec-rails#installation) if necessary) on the commits description?

File: Gemfile [hunk 2]
```diff
@@ -75,6 +75,7 @@ group :development do
   gem "listen",                   "~> 3.0.5"
   gem "spring",                   "~> 1.7.2"
   gem "spring-watcher-listen",    "~> 2.0.0"
+  gem "spring-commands-rspec",    "~> 1.0"
```
Comment (roalcantara) [inline Gemfile]: About the commit message: why is this necessary? What is this for?

File: Gemfile (hunk 1 above)
Comment (ddomingues) [inline Gemfile]: I don't understand your doubts. Take a look at my commit message `Move rspec-rails to correct group`, in this case, `test`. The gem was in the wrong group.

Then I just update to the stable version 🤗

File: Gemfile (hunk 2 above)
Comment (ddomingues) [inline Gemfile]: It is an addon to [spring](https://github.com/rails/spring) that speeds up the test runner through a bin stub, take a look for more details [here](https://github.com/jonleighton/spring-commands-rspec)

**Comment by ddomingues** [comment]
@roalcantara sorry, I just start some things and put here...

Questions:

1. Why is this a `chore` commit, and not a `docs`?
   You're right... I'm not using commitzen, so I don't remember all commands 😞 
2. How can we standardize the commits messages?
   We need to talk and to define it.
